gpii/node_modules/testing/src/CloudBasedOAuth2.js

```diff
@@ -134,9 +134,9 @@ gpii.test.cloudBased.oauth2.verifyAuthorizeResponse = function (body, authorizeR
 
 gpii.test.cloudBased.oauth2.sendLoginRequest = function (loginRequest, options) {
     var formBody = {username: options.username, password: options.password};
-    
+
     formBody = gpii.test.cloudBased.oauth2.filter(formBody, options, "loginRequestForm");
-    
```

Comment:
And there was me thinking from the notification email that you were approving of the incredible elegance of our privacy filtering system, rather than the removal of some whitespace :)

Perhaps it really is time for me to fork our jshint and grunt-jshint so that they can check this.........